slemicro/create_vm.sh

```diff
@@ -82,55 +91,71 @@ mkisofs -quiet -full-iso9660-filenames -o ${VMFOLDER}/ignition-and-combustion-${
 # Remove leftovers
 rm -Rf ${TMPDIR}
 
-# Create and launch the VM using UTM
-OUTPUT=$(
-osascript <<END
-tell application "UTM"
-	--- specify a boot ISO
-	set iso to POSIX file "${VMFOLDER}/ignition-and-combustion-${VMNAME}.iso"
-	-- specify the RAW file
-	set rawfile to POSIX file "${VMFOLDER}/${VMNAME}.raw"
-	--- create a new QEMU VM 
-	set vm to make new virtual machine with properties {backend:qemu, configuration:{cpu cores:${CPUS}, memory: ${MEMORY}, name:"${VMNAME}", architecture:"aarch64", drives:{{removable:true, source:iso}, {removable:false, source:rawfile}}}}
-	start vm
-	repeat
-		if status of vm is started then exit repeat
-	end repeat
-	get address of first serial port of vm
-end tell
+# if x86_64, convert the image to qcow2 to be used by kcli
+if [ $(uname -o) == "GNU/Linux" ]; then
+  qemu-img convert -O qcow2 ${VMFOLDER}/${VMNAME}.raw ${VMFOLDER}/${VMNAME}.qcow2
+  virt-install --name ${VMNAME} --autostart --noautoconsole --memory ${MEMORY} --vcpus ${CPUS} --disk ${VMFOLDER}/${VMNAME}.qcow2 --import --cdrom ${VMFOLDER}/ignition-and-combustion-${VMNAME}.iso --network default --osinfo detect=on,name=sle-unknown
+  echo "VM created. Waiting for IP..."
+  timeout=180
+  count=0
+  while [ $(virsh domifaddr ${VMNAME} | grep ipv | awk '{print $4}'| cut -d"/" -f 1 | wc -l) -ne 1 ]; do
+      count=$((count + 1))
+      if [[ ${count} -ge ${timeout} ]]; then
+        break
+      fi
+      sleep 1
+      echo -n "."
+  done
+  VMIP=$(virsh domifaddr ${VMNAME} | grep ipv | awk '{print $4}'| cut -d"/" -f 1 )
+else # Darwin
```

Comment:
```suggestion
elif [ $(uname -o) == "Darwin" ]; then
```
For consistency with the other `if`